### PR TITLE
Added long description for PyPI

### PR DESCRIPTION
**Pull request by ForestJay**
This seems to be the only change needed to upload our code into PyPI.

### CODE REVIEW

**Comment by mostafazh** [comment]
Looks like Travis is complaining about the following flake8 errors:

> affectiva/job.py:7:1: E302 expected 2 blank lines, found 1
> affectiva/job.py:84:1: W293 blank line contains whitespace
> setup.py:20:1: W293 blank line contains whitespace

File: affectiva/api.py [hunk 1]
```diff
@@ -149,26 +149,3 @@ def add_annotation(self, entry, source, key, value):
         resp = requests.post(entry['annotations'], auth=self._auth, headers=ACCEPT_JSON, data={"annotation[source]": source, "annotation[key]": key, "annotation[value]": value})
         resp.raise_for_status()
         return resp.json()
-
```
Comment (ForestJay) [inline affectiva/api.py]: Here is why this was moved:
Does not work old way:
ForestsAffectivaMacbook:Paula’sDad foresthandford$ ./upload_driver_videos.py
INFO:Video Upload:Preparing for upload!
INFO:Video Upload:Loaded data for participant 7
INFO:Video Upload:uploading /Users/foresthandford/Desktop/Affectiva Rocks!/Data Acquisition/In-car/Paula’sDad/2017_1109_212201_038B.MOV started
INFO:Video Upload:Uploading /Users/foresthandford/Desktop/Affectiva Rocks!/Data Acquisition/In-car/Paula’sDad/2017_1109_212201_038B.MOV completed
INFO:Video Upload:uploading /Users/foresthandford/Desktop/Affectiva Rocks!/Data Acquisition/In-car/Paula’sDad/2017_1109_212201_037A.MOV started
Traceback (most recent call last):
  File “./upload_driver_videos.py”, line 126, in <module>
    entry.add_representation(job[‘input’], road_path, ‘application/vnd.affectiva.road+mov’)
AttributeError: ‘Entry’ object has no attribute ‘add_representation’


[3:29] 
Works new way:
INFO:Video Upload:Preparing for upload!
INFO:Video Upload:Loaded data for participant 7
INFO:Video Upload:uploading /Users/foresthandford/Desktop/Affectiva Rocks!/Data Acquisition/In-car/Paula’sDad/2017_1109_211201_034B.MOV started
INFO:Video Upload:Uploading /Users/foresthandford/Desktop/Affectiva Rocks!/Data Acquisition/In-car/Paula’sDad/2017_1109_211201_034B.MOV completed
INFO:Video Upload:uploading /Users/foresthandford/Desktop/Affectiva Rocks!/Data Acquisition/In-car/Paula’sDad/2017_1109_211201_033A.MOV started
INFO:Video Upload:Uploading /Users/foresthandford/Desktop/Affectiva Rocks!/Data Acquisition/In-car/Paula’sDad/2017_1109_211201_033A.MOV completed
INFO:Video Upload:annotating https://index.affectiva.com/entries/3623685538543469618
INFO:Video Upload:Annotating /Users/foresthandford/Desktop/Affectiva Rocks!/Data Acquisition/In-car/Paula’sDad/2017_1109_211201_034B.MOV completed
INFO:Video Upload:uploading /Users/foresthandford/Desktop/Affectiva Rocks!/Data Acquisition/In-car/Paula’sDad/2017_1109_211701_036B.MOV started
INFO:Video Upload:Uploading /Users/foresthandford/Desktop/Affectiva Rocks!/Data Acquisition/In-car/Paula’sDad/2017_1109_211701_036B.MOV completed
INFO:Video Upload:uploading /Users/foresthandford/Desktop/Affectiva Rocks!/Data Acquisition/In-car/Paula’sDad/2017_1109_211701_035A.MOV started
INFO:Video Upload:Uploading /Users/foresthandford/Desktop/Affectiva Rocks!/Data Acquisition/In-car/Paula’sDad/2017_1109_211701_035A.MOV completed
INFO:Video Upload:annotating https://index.affectiva.com/entries/5018326886780487761

Comment (caboteria) [reply affectiva/api.py]: a better option would be to copy the code, not move it

Comment (caboteria) [reply affectiva/api.py]: an even better option would be to figure out why api.py and job.py appear to be two different ways of doing the same thing

**Pull request merged**